cinema/serializers.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class CinemaSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=False, max_length=255)
+    description = serializers.CharField(style={"bace_template": "textarea.html"})
```

Comment:
There is a typo in the `style` attribute for the `description` field. It should be `base_template` instead of `bace_template`.